imports/plugins/core/job-collection/lib/jobCollection.app-test.js

```diff
@@ -339,6 +352,7 @@ describe("JobCollection", function () {
   });
 
   it("should have dependent job saved after cancelled antecedent is also cancelled", function (done) {
+    this.timeout(15000);
```

Comment:
The changes I made seem to have changed the order in which Meteor runs the tests, and a bunch of them started consistently timing out on my machine and PR. I think the timeouts happen for whatever tests run first, because often the startup code (mostly migrations) is still running and ends up slowing down how fast the db ops in tests happen.

It seems like Meteor should have some better solution, like allowing us to indicate when startup code is done, and then starting the test run. Maybe if we moved the migrations and other things into `Meteor.startup` blocks? But anyway, bumping up the timeouts on every test that failed seemed the quickest way to unblock this.